sql/mysqld.cc

```diff
@@ -7004,11 +7008,24 @@ static int show_heartbeat_period(THD *thd, SHOW_VAR *var, char *buff,
   else
     var->type= SHOW_UNDEF;
   return 0;
-}
-
+}\
```

Comment:
I think this `\` was added by mistake, right?

Reply:
Yes - will remove it shortly